### ISSUE TITLE
Roblox's process still alive after closing the window

### ISSUE THREAD

**Issue by splatert** [vinegarhq/vinegar#387]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Closing Roblox does not end its process. You can find it on the process list still running.
In order to visit different places, I have to keep closing any running instances.

Steps to reproduce this issue:
1. Join a place
2. Leave it
3. Find Roblox still running in process list / task manager.

### System information

```shell
* Vinegar: v1.7.3 
* Distro: Freedesktop SDK 23.08 (Flatpak runtime)
* Processor: Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
  * Supports AVX: true
  * Supports split lock detection: false
* Kernel: 5.15.0-87-generic
* Wine (Player): v9.2 (Staging)*
* Wine (Studio): v9.2 (Staging)*
* Flatpak: [x]
* Cards:
  * Card 0: nvidia 0000:01:00.0 /sys/class/drm/card0
```

* Sysinfo mentions that I'm on Wine 9.2 Staging but I'm in fact using Wine GE.

**Comment by EDLLT** [comment]
> ### Acknowledgement of preliminary instructions
> 
>     * [x]  I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
> 
> 
> ### What problem did you encounter?
> 
> Closing Roblox does not end its process. You can find it on the process list still running. In order to visit different places, I have to keep closing any running instances.
> 
> Steps to reproduce this issue:
> 
>     1. Join a place
> 
>     2. Leave it
> 
>     3. Find Roblox still running in process list / task manager.
> 
> 
> ### System information
> 
> ```shell
> * Vinegar: v1.7.3 
> * Distro: Freedesktop SDK 23.08 (Flatpak runtime)
> * Processor: Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
>   * Supports AVX: true
>   * Supports split lock detection: false
> * Kernel: 5.15.0-87-generic
> * Wine (Player): v9.2 (Staging)*
> * Wine (Studio): v9.2 (Staging)*
> * Flatpak: [x]
> * Cards:
>   * Card 0: nvidia 0000:01:00.0 /sys/class/drm/card0
> ```
> 
>     * Sysinfo mentions that I'm on Wine 9.2 Staging but I'm in fact using Wine GE.

Faced the same issue back when I using flatpak
A source installation fixed the issue for me

**Comment by wisheth** [comment]
This is an issue that has persisted with me for a while. I use Flatpak

**Comment by EDLLT** [comment]
> Interesting. I faced the same issue on source and got it fixed via flatpak.

Interesting
When you were using a source installation, which wine prefix did you set?

Because in my case a source installation fixed the issue, I had my wine set to wine ge initially but after wael compiled vinegar's wine, I began using that instead and it fixed all of my issues(talking about the source installation here)

**Comment by Coder2195Text** [comment]
Turns out flatpak is also screwing up again, so it's both platforms. When I
was on source, I just went into flatpak files and copied their binaries for
wine stuff.

**Comment by apprehensions** [comment]
Roblox logs?

**Comment by apprehensions** [comment]
Fixed in ee9ef615b92888955ecdcfb569cc45e03df24637
